### ISSUE TITLE
[Protocol3] ABI generation error

### ISSUE THREAD

**Issue by dong77** [Loopring/protocols#315]
When I ran `solc --abi contracts/iface/*.sol -o ABI/version30/ --overwrite` I got the following errors:

```
contracts/iface/IBlockVerifier.sol:57:27: Error: Expected ',' but got identifier
        uint[18] calldata vk
                          ^^
contracts/iface/IDecompressor.sol:28:24: Error: Expected ',' but got identifier
        bytes calldata data
                       ^--^
contracts/iface/IExchange.sol:250:27: Error: Expected ',' but got identifier
        uint[30] calldata accountMerkleProof,
                          ^----------------^
contracts/iface/ILoopringV3.sol:64:17: Error: Expected ',' but got 'payable'
        address payable protocolFeeVault
                ^-----^
contracts/iface/IProtocolFeeVault.sol:57:17: Error: Expected ',' but got 'payable'
        address payable auctionAddr
```

Does anyone have the same issues?

**Comment by dong77** [comment]
@Brechtpd @kongliangzhong @letsgoustc

**Comment by Brechtpd** [comment]
genAbi doesn't use the npm version of solc (used for compiling the contracts) but the globally installed version. So you probably have the wrong version installed globally:

```
solc --version
```

Should return 0.5.7

**Comment by dong77** [comment]
I see. I fixed this in #316
